app/models/conversion_host.rb

```diff
@@ -56,6 +56,22 @@ def ipaddress(family = 'ipv4')
     resource.ipaddresses.detect { |ip| IPAddr.new(ip).send("#{family}?") }
   end
 
+  def get_cpu_limit
```

Comment:
Do we need `get_`, or can we just redefine the existing method?

Reply:
`Setting(s)`?

Reply:
I'm neither a big fan of `get_`. It's an attribute of ConversionHost, but should default to `Settings.transformation.limits.cpu_limit_per_host` which we implemented in a previous PR. Can we override an attribute with a method ?

Reply:
Yep, I think this is the way: http://joshfrankel.me/blog/overriding-activerecord-getters-with-the-same-attribute-name/